ephios/core/services/notifications/types.py

```diff
@@ -377,14 +377,27 @@ def get_actions(cls, notification):
 
     @classmethod
     def get_subject(cls, notification):
-        event = AbstractParticipation.objects.get(
+        participation = AbstractParticipation.objects.get(
             id=notification.data.get("participation_id")
-        ).shift.event
-        return _("Participation {state} for {event}").format(
-            state=AbstractParticipation.States.labels_dict()[
-                notification.data["participation_state"]
-            ],
-            event=event,
+        )
+        text = {
+            AbstractParticipation.States.CONFIRMED: _("{participant} signed up for {shift}"),
+            AbstractParticipation.States.REQUESTED: _(
```

Comment:
i think the present continous is not the correct form, either "has requested a participation for" or "has requested to participate in" would be better IMO